actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -10,8 +10,12 @@ def default_render(*args)
       if template_exists?(action_name.to_s, _prefixes, variants: request.variant)
         render(*args)
       else
-        logger.info "No template found for #{self.class.name}\##{action_name}, rendering head :no_content" if logger
-        head :no_content
+        if block_given?
+          yield(*args)
+        else
+          logger.info "No template found for #{self.class.name}\##{action_name}, rendering head :no_content" if logger
+          head :no_content
+        end
```

Comment:
I still think we should do it this way:

```ruby
def default_render(*args)
  if template_exists?(action_name.to_s, _prefixes, variants: request.variant)
    render(*args)
  else
    block_given? ? yield : default_missing_template_action
  end
end

def default_missing_template_action
  logger.info "No template found for #{self.class.name}\##{action_name}, rendering head :no_content"
  head :no_content
end

# In responders
def to_html
  default_render do
    navigation_behavior
  end
end

def navigation_behavior
  if get?
    controller.default_missing_template_action
  # Other branches...
  end
end
```

This way responders doesn't have to reimplement our convention on what happens when a template is missing. Do you agree with this, @rafaelfranca?